src/components/Calculator-class-based.js

```diff
@@ -0,0 +1,180 @@
+import React, { Component } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import '../scss/Calculator.scss';
+import calculate from '../logic/calculate';
+
+const buttons = [
+  {
+    symbol: 'AC',
+    id: uuidv4(),
```

Comment:
- [ ] I don't think we need this component. It is a duplicate of the functional component - except it is class-based. Kindly remove this component.